src/compiler/checker.ts

```diff
@@ -3497,16 +3501,20 @@ module ts {
             return globalESSymbolConstructorSymbol || (globalESSymbolConstructorSymbol = getGlobalValueSymbol("Symbol"));
         }
 
+        function createTypeFromGenericGlobalType(genericGlobalType: GenericType, elementType: Type): Type {

```

Comment:
Let's actually leave a comment because I'm still reading this wrong every time I look at it. Can you make the first parameter `globalType` and document that you're expecting it to take one type parameter?